### PR TITLE
Firewall objects

### PR DESCRIPTION
**Pull request by gsanchietti**
NethServer/nethsecurity#567

### CODE REVIEW

File: objects.rst [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+   - create, modify, or delete host sets without directly editing firewall rules
+   - changes to a host set automatically apply to all rules using that set
+
+4. Use cases:
+
+   - group company servers for access control
+   - create allow or deny lists for specific network segments
+   - manage remote access for multiple VPN users
+
```
Comment (cotosso) [inline objects.rst:127]:
```suggestion

.. note::

Host sets are fully supported in their expressive completeness (IP, CIDR, range, groupings) within firewall rules. Other panels might only support a reduced subset (for example, MultiWAN only supports single IP addresses and CIDR). In such cases, only compatible host sets will be displayed.
  
```

File: objects.rst [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+Use cases for domain sets:
+
+- application control: manage access to cloud services or social media platforms
+- security: create allow rules for trusted domains
+- malware prevention: create deny rules for known malicious domains
+
+Benefits of using domain sets:
+
+- simplifies management of rules based on web addresses
```
Comment (filippocarletti) [inline objects.rst:237]:
```suggestion
- simplify management of rules based on web addresses
```

File: objects.rst [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+- automatically handles IP address changes of websites
```
Comment (filippocarletti) [inline objects.rst:238]:
```suggestion
- automatically handle IP address changes of websites
```

File: objects.rst [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+- reduces the need for manual updates to firewall rules
```
Comment (filippocarletti) [inline objects.rst:239]:
```suggestion
- reduce the need for manual updates to firewall rules
```

File: objects.rst [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+- provides a more intuitive way to control access to web-based services
```
Comment (filippocarletti) [inline objects.rst:240]:
```suggestion
- provide a more intuitive way to control access to web-based services
```

File: objects.rst [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+
+   - Access the ``Objects`` page under the ``User and objects`` section from the left sidebar menu
+   - Navigate to the ``Domain sets`` tab
+   - Click on :guilabel:`Add somain set` button
+
+2. Enter the Domain Set name:
+
+   - In the ``Name`` field, enter a descriptive name for your domain set
+   - Use only letters and numbers; spaces and special characters are not allowed
+   - Choose a name that clearly identifies the purpose or group of domains
```
Comment (filippocarletti) [inline objects.rst:273]:
```suggestion
   - Choose a name that clearly identifies the purpose of the group of domains
```

**Review by filippocarletti — CHANGES_REQUESTED** [review]

File: objects.rst [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+
+3. Select IP version
+
+   - Under ``IP version``, choose between IPv4 and IPv6
+   - Select IPv4 for standard internet protocol addresses
+   - Choose IPv6 if you're using the newer, expanded address format
+
+4. Add Records
+
+   - In the ``Records`` field, you can add the hosts for this set
```
[the remark this answers is not in the document]
Comment (gsanchietti) [reply objects.rst]: Not sure about this

[the remark this answers is not in the document]
Comment (filippocarletti) [reply objects.rst]: What about "to"?

[the remark this answers is not in the document]
Comment (gsanchietti) [reply objects.rst]: Yes, maybe "to" is better. But I still prefer the original form

[the remark this answers is not in the document]
Comment (filippocarletti) [reply objects.rst]: Let's go with "for", then.

[the remark this answers is not in the document]
Comment (gsanchietti) [reply objects.rst]: :ok:

[the remark this answers is not in the document]
Comment (gsanchietti) [reply objects.rst:177]: Fixed.

**Pull request merged**